.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
```

Comment:
Do we need so many? OKay, let it be. Nice one, thanks.

Reply:
this all comes with the create react app itself.

Reply:
Really??? Okay, didn't notice LoL.